gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -152,42 +152,26 @@ def compute_hard_filters(cov_threshold: int) -> hl.Table:
 
 
 def compute_sex() -> hl.Table:
-    # Compute sex chrom poloidy
-    ht = impute_sex_ploidy(
-        get_gnomad_v3_mt(remove_hard_filtered_samples=False),
-        excluded_calling_intervals=telomeres_and_centromeres.ht()
+    mt = get_gnomad_v3_mt(
+        key_by_locus_and_alleles=True,
+        remove_hard_filtered_samples=False,
     )
-    ht = ht.checkpoint('gs://gnomad-tmp/sex_depth.ht', overwrite=True)
+    mt = mt.filter_rows((hl.len(mt.alleles) == 2))
```

Comment:
I don't think you need this filter, since you want to compute depth from non ref sites and ref blocks when imputing ploidies